src/tracking.rs

```diff
@@ -114,7 +116,7 @@ pub(crate) async fn download_gossip(persistence_sender: mpsc::Sender<GossipMessa
 
 			if needs_to_notify_persister {
 				needs_to_notify_persister = false;
-				persistence_sender.send(GossipMessage::InitialSyncComplete).await.unwrap();
+				completion_sender.send(()).await.unwrap();
```

Comment:
couldn't this be up to 100 gossip messages behind? Are we sure we don't want the persistence criterion to be determinant in triggering the initial snapshot?